Examples/create-windows-cloud-image.ps1

```diff
@@ -12,39 +12,70 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-$ErrorActionPreference = "Stop"
```

Comment:
please leave as much as possible the code from the previous way as you can. (ditto1)